src/local-fs.ts

```diff
@@ -72,15 +72,19 @@ export class LocalFileSystem implements FileSystem {
                             type: 'fileCreated',
                             fullPath: relPath.split(path.sep).join(pathSeparator),
                             newContent: content
-                        })));
+                        }))
+                        .catch(() => {})
```

Comment:
@amir-arad we need that also for cache-fs, or it should know about local-fs can emmit error event, or to have error handler, but problem is cache-fs don't know which fs it caches, thats also why I was thinking the error event is good idea.